aws_lambda_builders/workflows/nodejs_npm/workflow.py

```diff
@@ -55,8 +55,13 @@ def __init__(self,
 
         npm_install = NodejsNpmInstallAction(artifacts_dir,
                                              subprocess_npm=subprocess_npm)
+
+        npm_copy_npmrc = NodejsNpmrcCopyAction(tar_package_dir, source_dir, osutils=osutils)
+
         self.actions = [
             npm_pack,
+            npm_copy_npmrc,
```

Comment:
Does the order matter at all? Does `npm_copy_npmrc` just need to be between `npm_pack` and 'npm_install` or must is happen before the copy source?

Trying to understand if there is some documentation we need to add here for the future.

Reply:
it needs to come between npm_pack and npm_install because `npm pack` will exclude it from the package, but it's needed for `npm install` to correctly function (it may contain URLs to module repositories etc)